client/src/helpers/APICalls/profile.ts

```diff
@@ -0,0 +1,15 @@
+import { FetchOptions } from '../../interface/FetchOptions';
+
+const getProfileDetails = async( profile_id: string ) => {
+    const fetchOptions: FetchOptions = {
+        method: 'GET',
+        credentials: 'include',
+      };
+      return await fetch(`/profiles/search?_id=${profile_id}`, fetchOptions)
```

Comment:
As discussed on our call today, we need to eventually refactor the route to take path parameters as opposed to query parameters (maybe in another PR)